composer.json

```diff
@@ -32,7 +32,8 @@
         "symfony/polyfill-intl-idn": "^1.10",
         "symfony/polyfill-mbstring": "~1.0",
         "symfony/polyfill-php72": "~1.5",
-        "symfony/polyfill-php73": "^1.11"
+        "symfony/polyfill-php73": "^1.11",
+        "symfony/google-mailer": "4.3.x-dev"
```

Comment:
this should be removed, the package is already "replaced" in this composer.json file